src/Apps/Auction/Routes/AuctionConfirmRegistrationRoute.tsx

```diff
@@ -123,34 +127,30 @@ const AuctionConfirmRegistrationRoute: React.FC<AuctionConfirmRegistrationRouteP
             <Form>
               <Join separator={<Spacer y={2} />}>
                 {needsIdentityVerification ? (
-                  <IdentityVerificationWarning />
+                  <IdentityVerificationWarning
+                    additionalText={additionalText}
+                  />
                 ) : (
+                  <ConditionsOfSaleMessage additionalText={additionalText} />
+                )}
+
+                {!hasPhoneNumber && (
                   <GridColumns>
-                    {!hasPhoneNumber ? (
```

Comment:
does it matter that this `GridColumns` previously got rendered every time?

Reply:
Previously there was a conditional rendering but now I think `GridColumns` can be removed.